commands/service_deploy.go

```diff
@@ -32,6 +35,7 @@ To get more information, see the [deploy page from the documentation](https://do
 		RunE:    c.runE,
 		Args:    cobra.MaximumNArgs(1),
 	})
+	c.cmd.Flags().BoolVarP(&c.force, "force", "f", c.force, "Force deploy the service by overwriting existing service with the same alias if there is")
```

Comment:
In one command we have `yes` , here we have `force` so it's starting to mix,

maybe change to overwrite?

Also,

`Force deploy the service by overwriting existing service with the same alias if there is` to
`Deploying overwrites existing service with the same alias`

One more thing related to `all` (it's not about this PR but in general) in service delete we have:
`	c.cmd.Flags().BoolVarP(&c.yes, "yes", "y", c.yes, `Automatic "yes" to all prompts and run non-interactively`)`
But there is no information for the user what question will be asked during the process and it isn't possible to answer to one question yes and others no by using the flags.

Reply:
* 'Force' makes a good sense to me, it's also widely used. It has a meaning about 'force doing' things that aren't meant to be doable when it's not used. I'm not against to overwrite as well but  force is a bit better to me and we can use it in other commands so it's more convenient. 

* 'yes' and 'force' has totally meanings and results.

>But there is no information for the user what question will be asked during the process and it isn't possible to answer to one question yes and others no by using the flags.

* This is why 'yes' command is good. Devs shouldn't use it if they didn't experienced the questions before. So have to use the command without 'yes' and once they get how it's work they can switch to 'yes' to skip interactive.